packages/frontend/src/components/table/ProjectCell.tsx

```diff
@@ -51,6 +59,26 @@ export function ProjectCell({ project, type, highlightL2 }: ProjectCellProps) {
           </span>
         </span>
       )}
+      {project.isArchived && (
+        <span className="relative pl-1.5">
+          <span
+            className="Tooltip ml-1.5 inline-block"
+            title={'This project is archived and no longer maintained.'}
+          >
+            <ArchivedIcon className="absolute top-px left-1.5 h-4" />
+          </span>
+        </span>
+      )}
+      {project.isUpcoming && (
+        <span className="relative pl-3">
+          <span
+            className="Tooltip ml-0.5 inline-block"
+            title={'This is an upcoming project. Stay tuned!'}
+          >
+            <UpcomingIcon className="absolute top-[-1px] left-0.5 h-4" />
+          </span>
+        </span>
```

Comment:
```suggestion
        <span className="pl-1.5">
          <span
            className="Tooltip inline-block"
            title={'This is an upcoming project. Stay tuned!'}
          >
            <UpcomingIcon className="relative top-px h-4" />
          </span>
        </span>
```

Keep it simple. Copying styles was way to go here